acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/ValidatorSlashingDetectionAcceptanceTest.java

```diff
@@ -0,0 +1,474 @@
+        UInt64.valueOf(slashedValidatorIndex),
+        slashedValidatorKeyPair.getSecretKey(),
+        firstTekuNode,
+        slashingEventType);
+
+    final TekuNode secondTekuNode =
+        createTekuNode(
+            config ->
+                configureNode(config, genesisTime, network)
+                    .withAltairEpoch(altairEpoch)
+                    .withStopVcWhenValidatorSlashedEnabled()
+                    .withInteropValidators(32, 32)
+                    .withPeers(firstTekuNode));
+
+    secondTekuNode.start();
+
+    secondTekuNode.waitForLogMessageContaining(
+        String.format(slashingActionLog, slashedValidatorKeyPair.getPublicKey().toHexString()));
+
+    firstTekuNode.stop();
+  }
+
+  @ParameterizedTest
+  @MethodSource("getSlashingEventTypes")
+  void shouldShutDownWhenOwnedValidatorSlashed_StandAloneVC_MultiplePeers(
+      final SlashingEventType slashingEventType) throws Exception {
+
+    final int genesisTime = timeProvider.getTimeInSeconds().plus(10).intValue();
+    final UInt64 altairEpoch = UInt64.valueOf(100);
+
+    final TekuNode firstTekuNode =
+        createTekuNode(
+            config ->
+                configureNode(config, genesisTime, network)
+                    .withRealNetwork()
+                    .withAltairEpoch(altairEpoch)
+                    .withInteropValidators(0, 32));
+
+    firstTekuNode.start();
+
+    firstTekuNode.waitForEpochAtOrAbove(1);
+
+    final TekuNode secondBeaconNode =
+        createTekuNode(
+            config ->
+                configureNode(config, genesisTime, network)
+                    .withRealNetwork()
+                    .withAltairEpoch(altairEpoch)
+                    .withPeers(firstTekuNode));
+
+    final TekuValidatorNode secondValidatorClient =
+        createValidatorNode(
+            config ->
+                config
+                    .withNetwork("auto")
+                    .withValidatorApiEnabled()
+                    .withStopVcWhenValidatorSlashedEnabled()
+                    .withInteropValidators(32, 32)
+                    .withBeaconNode(secondBeaconNode));
+
+    secondBeaconNode.start();
+
+    secondValidatorClient.start();
+
+    firstTekuNode.waitForEpochAtOrAbove(2);
+
+    final int slashedValidatorIndex = 34;
+    final BLSKeyPair slashedValidatorKeyPair = getBlsKeyPair(slashedValidatorIndex);
+
+    postSlashing(
+        UInt64.valueOf(60),
+        UInt64.valueOf(slashedValidatorIndex),
+        slashedValidatorKeyPair.getSecretKey(),
+        firstTekuNode,
+        slashingEventType);
+
+    secondValidatorClient.waitForLogMessageContaining(
+        String.format(slashingActionLog, slashedValidatorKeyPair.getPublicKey().toHexString()));
+
+    secondBeaconNode.stop();
+    secondValidatorClient.stop();
+  }
+
+  @ParameterizedTest
+  @MethodSource("getSlashingEventTypes")
+  void
+      shouldShutDownWhenOwnedValidatorSlashed_StandAloneVC_MultiplePeers_SlashingEventsThroughGossipOnly_NoBlocks(
```

Comment:
I don't understand why it's `NoBlocks` case, both nodes will have all blocks in this case. Protection activation could be the result of either gossip or block in this case.

Reply:
This is "no blocks" because the first node is not running any validator and hence won't produce any block. The slashing post is sent to that same node which is then gossiped...

Reply:
But is there any difference when node is block producer or just received block via gossip for this protection?

Reply:
If we'd like to simulate the slashing being sent within a block and not the slashing gossip topics, the node which receives the slashing should produce a block in order to include the slashing in it.
It goes like the following, assuming the slashed validator is run by Node 2
- Block scenario: Post slashing to Node 1 -> Node 1 produces a block including the slashing -> Node 1 gossip the block -> Node 2 receives the blocks and sees the slashing -> Node 2 shut down
- Proposer slashing or attester slashing gossip messages: Post slashing to Node 1 -> Node 1 sends the slashing message to the proposer or attester slashing gossip topic -> Node 2 receives the proposer or attester slashing message through gossip -> Node 2 shut down